### ISSUE TITLE
view background resource rename 

### ISSUE THREAD

**Issue by Amoi** [ticketmaster/Android-SecureEntrySDK#5]
Hi,

We've noticed that in the secureEntryView,  some of the ImageViews use the background called "background".   it can be a issue that when the consumer App have a drawable with the same name. It will be override by the consumer App one. 

Could you rename the resource and make it like "tm_background" or something like that?
It will help not to pollute the whole namespace.

Thanks

**Comment by kylejablonski** [comment]
Yes, I can provide this in the next release of the library. I will drop a comment here when it is available. Sorry for this inconvenience.

**Comment by Amoi** [comment]
Thanks a lot
